content/news/021/index.md

```diff
@@ -1018,13 +1018,11 @@ Want something mentioned in the next newsletter?
 Also, subscribe to [@rust_gamedev on Twitter][@rust_gamedev]
 or [/r/rust_gamedev subreddit][/r/rust_gamedev] if you want to receive fresh news!
 
-<!--
-TODO: Add real links and un-comment once this post is published
 **Discuss this post on**:
-[/r/rust](TODO),
-[Twitter](TODO),
+[/r/rust](https://reddit.com/r/rust/comments/n8g7nc/this_month_in_rust_gamedev_21_april_2021),
```

Comment:
:thinking:  i guess one reddit link should be enough? /r/rust_gamedev is the main one so I'd remove the link to /r/rust crosspost

Reply:
Fixed!